test/software/amazon/ion/impl/LocalSymbolTableImportAdapterTest.java

```diff
@@ -52,8 +54,7 @@ public void testMaxIdNoImports()
     }
 
     @Test
-    public void testMaxIdWithImports()
-        throws Exception
+    public void testGetMaxId()
```

Comment:
Consider adding `@Override` to the tests that are overridden from the abstract test class.